### DISCUSSION TITLE
Does autogluon support mix precision?

### DISCUSSION THREAD

**Discussion by talhaanwarch** [autogluon/autogluon#1131] [Q&A]
I want to know does autogluon support mix precision like tensorflow and pytorch?

**Comment by Innixma** [comment]
For Tabular we support any valid numpy / pandas dtypes, so you can specify the dtypes in your DataFrame and it should be respected. We automatically try to minimize memory usage, and individual models will further do optimizations during training that may include mixed precision.

**Reply by talhaanwarch** [reply]
what about imageprediction, can ag train model on  float16

**Reply by Innixma** [reply]
@zhreshold ?
